src/Line.js

```diff
@@ -536,8 +538,11 @@ export function Line(opts, data) {
 		});
 
 		series.forEach((s, i) => {
-			if (i > 0 && s.show)
-				drawPath(i);
+			if (i > 0 && s.show) {
+                drawPath(i);
+                fire("drawSeries", s, i);
```

Comment:
this should just be `fire("drawSeries", i);` the `s` can already be pulled from `self.series.y[i+1]`